src/utils/utils.ts

```diff
@@ -1,3 +1,9 @@
+// Add the interface for Spryker certifications
+interface SprykerCertifications {
```

Comment:
isnt this already part of the types file?

Reply:
No, they serve different purposes, but I'll clarify their roles:

In types.ts, SprykerCertifications is an interface that defines the shape/structure of the certification data
In utils.ts, the hasSprykerCertifications is a utility function that checks if any certification is true

Reply:
for me it looked like the interface itself was there 2 times. but not anymore